<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1294</width>
    <height>754</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="ocr">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>OCR error</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="label_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>381</width>
     <height>531</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="metadata">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>341</width>
     <height>201</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="picture_frame">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>611</width>
     <height>441</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="ruler_noise">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>120</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Ruler noise
(OCR)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="handwritten_noise">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>120</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Handwritten
INHS ID 
noise (OCR)</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="scientific_name">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>260</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>230</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Actual Name (metadata wrong):</string>
   </property>
  </widget>
  <widget class="QPushButton" name="true_error">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>10</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>True
metadata
error</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tilt">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>10</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Tilt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="synonym">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>120</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Potential
synonym</string>
   </property>
  </widget>
  <widget class="QPushButton" name="complex_name_format">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>10</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Complex
name
format</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inadmissibility">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>120</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Inadmissibility</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="further_descr">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>350</y>
     <width>241</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>320</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Further Description:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ok">
   <property name="geometry">
    <rect>
     <x>1230</x>
     <y>10</y>
     <width>61</width>
     <height>211</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="sys_status">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>560</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>530</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>System Status:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
